src/EDMF_Environment.jl

```diff
@@ -93,44 +93,53 @@ function microphysics(en_thermo::SGSQuadrature, grid::Grid, state::State, precip
     inner_src = zeros(src_len)
     outer_src = zeros(src_len)
     i_ql, i_qi, i_T, i_cf, i_qt_sat, i_qt_unsat, i_T_sat, i_T_unsat = 1:env_len
-    i_SH_qt, i_Sqt_H, i_SH_H, i_Sqt_qt, i_Sqt, i_SH, i_Sqr, i_Sqs = 1:src_len
+    i_SH_qt, i_Sqt_θ, i_SH_θ, i_Sqt_qt, i_Sqt, i_SH, i_Sqr, i_Sqs = 1:src_len
+
```

Comment:
```suggestion
i_Sθ_qt, i_Sqt_θ, i_Sθ_θ, i_Sqt_qt, i_Sqt, i_Sθ, i_Sqr, i_Sqs = 1:src_len
```
We should replace h with θ everywhere.

Reply:
To be consistent with the rest of the code it should actually be  `θ_liq_ice` and not just `θ`. Same for `qt` it should be `q_tot`. But that would make some of the lines here quite involved with underscores as we go through `inner` and `outer`, `sat` and `unsat` options. So maybe it makes sense to drop `_liq_ice` notation inside the quadratures loop.